waspc/cli/exe/Main.hs

```diff
@@ -179,6 +182,7 @@ printUsage =
         cmd   "    clean                 Deletes all generated code, all cached artifacts, and the node_modules dir.",
               "                          Wasp equivalent of 'have you tried closing and opening it again?'.",
         cmd   "    build                 Generates full web app code, ready for deployment. Use when deploying or ejecting.",
+        cmd   "    build start           Previews the built app locally.",
```

Comment:
Now this command feels like it wants to be called `build preview`

Reply:
Closing in favor of https://github.com/wasp-lang/wasp/pull/2796#discussion_r2169355292